### ISSUE TITLE
aria-errormessage require aria-live in AXE, should be optional

### ISSUE THREAD

**Issue by DavidMacDonald** [dequelabs/axe-core#680]
Axe is flagging error messages requiring live regions.

```
<input type="text" id="LastName" 
class="req-string nameTxt datas" name="LastName" value="" aria-errormessage="last">

 <span id="last>... error message </span>
```

AXE flags and says
To solve this violation, you need to:
Fix the following:
aria-errormessage values `l `a `s `t` must use a technique to announce the message (e.g., aria-live, aria-describedby, role=alert, etc.)

The aria spec makes [live regions optional](https://www.w3.org/TR/wai-aria-1.1/#aria-errormessage).

"Authors **MAY** use live regions for the error message element applying either an aria-live property or using one of the live region roles, for example, alert. "

If someone hit submit and 15 error messages like this come up you don't want aria-live. There should be just one general message i.e., errors on page, that would have the live region.

**Comment by WilcoFiers** [comment]
This is intentional. See #499 for details. Until there is broad support for `aria-errormessage` aXe will expect that you use a fallback.
